clang/lib/Sema/SemaChecking.cpp

```diff
@@ -1498,6 +1498,24 @@ static void builtinAllocaAddrSpace(Sema &S, CallExpr *TheCall) {
   TheCall->setType(S.Context.getPointerType(RT));
 }
 
+static bool checkBuiltinInferAllocToken(Sema &S, CallExpr *TheCall) {
```

Comment:
Should it have test like Sema/arithmetic-fence-builtin.c or Sema/builtin-allow-runtime-check.c?

It would be easier to notice missing test coverage with smaller patches.